src/pages/blog/starlingx-harbor.md

```diff
@@ -0,0 +1,255 @@
+---
+templateKey: blog-post
+title: Enhancing StarlingX through Harbor as a Secure Container Registry Solution
+author: Jagatguru Mishra
+date: 2024-09-30T01:32:05.627Z
+
+category: 
+  - label: Features & Updates
+    id: category-A7fnZYrE1
+---
+Learn about using Harbor as a container registry in StarlingX 10.0. Harbor enhances StarlingX’s capabilities by offering robust, secure management of container images and cloud-native artifacts. <!-- more -->
+
+# About Harbor
+[Harbor](https://goharbor.io/ ) is an open-source container registry that enhances security by implementing policies and role-based access control. It ensures images are vulnerability-free through scanning and provides image signing for trust verification.
+
+With the 2.0 release, Harbor becomes the first OCI (Open Container Initiative)-compliant open-source registry capable of storing a wide range of cloud-native artifacts, including container images, Helm charts, OPAs, and Singularity, among others. Helm charts can now be pushed directly to Harbor using Helm3, eliminating the need for separate hosting in ChartMuseum. Instead, they are stored alongside container images under artifacts. As shown in the figure below, container images, Helm charts, and Cloud Native Application Bundles (CNAB) are hosted within the same project.
+

```

Comment:
The above paragraph is referring to an image, but there isn't one linked in here.

Reply:
This is a generic description explaining that the registry can store container images. An image file is not required for this context.

Reply:
I meant this note in the text: "As shown in the figure below". This implies that there is a picture below to look at, but there is no picture under this paragraph.

Reply:
Thanks @ildikov. I missed this completely. Removed the sentence.